spec/jobs/segment_identify_job_spec.rb

```diff
@@ -23,6 +23,7 @@
             hosting_type: 'self',
             version: Utils::VersionService.new.version.version.number,
             organization_name: membership.organization.name,
+            email: membership.user.email,
```

Comment:
Can you remove test line 46? 🙏 

`it 'includes the email of the membership' do`

Reply:
oh yes nice catch!